src/enclave-agent/src/services/images.rs

```diff
@@ -111,6 +99,22 @@ impl ImageService {
 
         Ok(image.to_owned())
     }
+
+    fn get_container_id(&self, req: &image::PullImageRequest) -> Result<String> {
+        let cid = req.get_container_id().to_string() ;
+        
+        if cid.is_empty() {
```

Comment:
empty `cid` is also detected by `verify_id` so maybe this check is not needed?

Reply:
Sure.

Reply:
the formatting of the test cases looks strange. maybe we want to add a case for an empty container id as well?